src/main/java/io/vertx/core/net/impl/KeyStoreHelper.java

```diff
@@ -236,6 +242,20 @@ public X509KeyManager getKeyMgr(String serverName) {
     return getKeyMgrFactory().getKeyManagers();
   }
 
+  public TrustManagerFactory getTrustMgr(String serverName)  {
+    try {
+      KeyStore trustStore = trustMgrMap.get(serverName);
```

Comment:
I would prefer this code to be done in the `KeyStoreHelper` constructor and keep a `Map<String, TrustManagerFactory >` so can be detected at this moment and this is also like how we do for the SNI key manager mapping.

Reply:
thats good

Reply:
is it ok now?